### DISCUSSION TITLE
Create User Account

### DISCUSSION THREAD

**Discussion by yannick2801** [Ysurac/openmptcprouter#2694] [Q&A]
Hi, 

Is it possible to create a users account, for the web interface, so the user cannot modify the parameters in the web interface, if he does not have the admin account.

Thanks
BR
Yannick

**Comment by ratsalad** [comment]
System/ACL list is theoretically what is supposed to offer that functionality if it works as intended.

**Reply by yannick2801** [reply]
Thanks, I have had a look but it doesn't seem to work properly.

**Reply by Ysurac** [reply]
The readonly part is not working, only access or deny is working for now.

**Reply by yannick2801** [reply]
ok, now it seems better. Thanks for help.

BR
